lib/money/rates_store/memory.rb

```diff
@@ -14,11 +14,13 @@ module RatesStore
     #   store.each_rate {|iso_from, iso_to, rate| puts "#{from} -> #{to}: #{rate}" }
     class Memory
       INDEX_KEY_SEPARATOR = '_TO_'.freeze
+      INDEX_KEY_DATE_SEPARATOR = '_AT_'.freeze
```

Comment:
btw, there's absolutely no restriction to using string keys, can be an array, especially since the store itself (@rates) can not be accessed directly

Reply:
Ah, really interesting idea. I'll look into changing that.